docs/developer/local_up_kubeedge.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Kubeedge Local Up
+sidebar_position: 10
+---
+
+This page explains how to use shell script **hack/local-up-kubeedge.sh** in **kubeedge/kubeedge** repo to create a local KubeEdge cluster, installing necessary components, and starting CloudCore and EdgeCore services.
+
+## Usage Steps
+
+1. Preparation:
+Ensure that you have kind, kubectl, docker, and other necessary tools installed.
+Ensure that your system has a Go language development environment installed, and the version meets the requirements of KubeEdge.
+Based on the user's choice of container runtime (docker, cri-o, isulad), user can verify the prerequisites via the script below, or it will also be verified before the installation start in **hack/local_up_kubeedge.sh**
+
+```shell
+verify_docker_installed
+verify_crio_installed
+verify_isulad_installed
+kubeedge::golang::verify_golang_version
+check_kubectl
+check_kind
+```
+
+2. Executing the Script:
+Before running the script, you can modify some environment variables to suit your needs, such as **LOG_DIR** and **CONTAINER_RUNTIME** in the script.
+The script should be run with administrative privileges, as some operations (e.g., software installation, system configuration changes) require sudo access.
+
+```shell
+hack/local_up_kubeedge.sh
+```
```

Comment:
the name of script is local-up-kubeedge.sh

Reply:
correct the script name